e2e/stories/rtc.e2e.ts

```diff
@@ -0,0 +1,152 @@
+import { Subscription } from "rxjs";
+import { RTCMessageSchema } from "../lib/rtc";
+import { cleaning, dom, init } from "../teardowns";
+
+// tslint:disable-next-line:no-var-requires
+const joiAssert = require("joi-assert");
+
+jest.setTimeout(15000); // for the unstable internet connection
+
+beforeAll(async () => init());
+
+afterAll(cleaning);
+
+describe("Real Time Communication", () => {
+
+  let RTCMessage: any;
+  let subscription: Subscription;
+
+  let recordId: string;
+
+  describe("create one record message", () => {
+
+    afterAll(() => {
+      subscription.unsubscribe();
+      RTCMessage = undefined;
+    });
+
+    it("should be received when new record inserted", (done) => {
+      subscription = dom.dataset("test_dataset")
+        .watch("created")
+        .subscribe((message) => {
+          RTCMessage = message;
+          expect(RTCMessage).toBeDefined();
+          done();
+        });
+
+      dom.dataset("test_dataset")
+        .insert([{test_field: "name"}])
+        .execute();
+    });
+
+    it("should have correct schema", () => {
+      joiAssert(RTCMessage, RTCMessageSchema);
+    });
+
+    it("should contain the same record as being inserted", () => {
+      expect(RTCMessage.data[0].test_field).toEqual("name");
+    });
+  });
+
```

Comment:
Just as a suggestion, we can also have tests to check that event is not received when we unsubscribe.

Reply:
I don't know how to do that because if we cancel subscription we will have no ways to check that events are not coming anymore :(